build.gradle

```diff
@@ -26,8 +26,8 @@ buildscript {
     classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.4'
     classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     classpath 'com.android.tools.build:gradle:1.5.0'
-    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
-//    classpath "com.athaydes.gradle.ceylon:ceylon-gradle-plugin:1.1"
+//    classpath 'com.redhat.ceylon.gradle:android:0.0.1-SNAPSHOT'
```

Comment:
@bjansen yep, I realized that :) I think the most reliable solution is to figure out which **packages** need recompilation... for large projects where incremental compilation helps, this would probably be quite enough.